docs/limitations.md

```
@@ -141,35 +158,158 @@ version of PostgreSQL or from lower version to higher version.
 `replicate_ddl_command()`.
 
-### Delta-Apply Column Requirements
+## Delta-Apply Column Requirements
 
-Columns configured for Delta-Apply conflict resolution **must** have a `NOT NULL`
+Columns configured for Delta-Apply conflict resolution must have a `NOT NULL`
 constraint. If a NULL value is encountered during delta application, the apply
 worker will error with:
 
-`ERROR: delta apply column can't operate NULL values`
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)